--- Pods/AppAuth/Source/iOS/OIDExternalUserAgentCatalyst.h	2020-05-18 20:33:58.000000000 +0300
+++ -	2020-05-18 20:47:53.000000000 +0300
@@ -25,7 +25,9 @@
 /*! @brief A Catalyst specific external user-agent that uses `ASWebAuthenticationSession` to
        present the request.
 */
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
 API_AVAILABLE(macCatalyst(13)) API_UNAVAILABLE(ios)
+#endif
 @interface OIDExternalUserAgentCatalyst : NSObject<OIDExternalUserAgent>
 
 /*! @internal
